hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1614,11 +1610,6 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, fmt.Errorf("failed to reconcile machine config server: %w", err)
 	}
 
-	// Reconcile the network policies
```

Comment:
If we just do this how would this operator reconcile policies in < 4.15? We'd need this to be optional and driven by a cpo image label that specify its ability to reconcile networks, see e.g. controlPlaneOperatorAppliesManagementKASNetworkPolicyLabel